### ISSUE TITLE
Missing support for non @model directive types

### ISSUE THREAD

**Issue by michalisAsp** [aws-amplify/amplify-android#380]
I have a schema with non model directive types. `amplify codegen models` generates an empty class Post.java. 

```
type Blog @model {
     id: ID!
     post: Post
}

type Post {
    description : String
}
```

amplify version 4.18.0
amplify android version 0.10.0

**related issues**
Seems to be supported for iOS, but not for Android yet.
https://github.com/aws-amplify/amplify-cli/issues/3506
https://github.com/aws-amplify/amplify-cli/issues/2988

**Comment by jamesonwilliams** [comment]
Hi @michalisAsp, thanks for creating the issue. This is missing functionality on Android that we are aware of, internally. That said, we didn't actually have a GitHub tracking the feature, until now. We will be building support for this in the near-term. Will update this issue when there are developments.

**Comment by jpignata** [comment]
This was included in our 1.0.0 release. Please let us know your feedback!
